pensjon-brevbaker/src/main/kotlin/no/nav/pensjon/brev/template/render/PensjonLatexRenderer.kt

```diff
@@ -184,15 +214,18 @@ object PensjonLatexRenderer : LetterRenderer<RenderedLatexLetter>() {
             }
         }
 
-    private fun LatexAppendable.renderParagraphContent(scope: ExpressionScope<*, *>, element: Element.OutlineContent.ParagraphContent<*>): Unit =
+    private fun LatexAppendable.renderParagraphContent(scope: ExpressionScope<*, *>, element: ParagraphContent<*>): Unit =
         when (element) {
-            is Element.OutlineContent.ParagraphContent.Form -> renderForm(scope, element)
-            is Element.OutlineContent.ParagraphContent.Text -> renderTextContent(scope, element)
-            is Element.OutlineContent.ParagraphContent.ItemList -> renderList(scope, element)
-            is Element.OutlineContent.ParagraphContent.Table -> renderTable(scope, element)
+            is ParagraphContent.Form -> renderForm(scope, element)
+            is Text -> renderTextContent(scope, element)
```

Comment:
Veldig flisespikkeri, men det kunne godt vært likt mellom alle `is` her - altså at alle har lik static import nivå.

Reply:
fixed